assets/license-header

```diff
@@ -0,0 +1,15 @@
+<https://github.com/radicle-dev/<project name>>
```

Comment:
omg seriously? All the information needs to be included in _every_ source file?

Reply:
Maybe not. Checking with the legals :sweat_smile:

Reply:
Ok, so @angedupre is going to give us the copy that doesn't include all of this and looks more like the `libgit2` one you shared earlier :+1:

Reply:
Yep! Created some condensed copy for @FintanH for inclusion at the top of the source files. As mentioned in a comment above - there was some confusion on my side but I think we're clear now and have copy that makes sense.